universal-application-tool-0.0.1/app/models/Applicant.java

```diff
@@ -1,35 +1,42 @@
 package models;
 
-import io.ebean.annotation.DbJsonB;
-import io.ebean.text.json.EJson;
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import io.ebean.annotation.DbJson;
 import java.io.IOException;
-import java.util.Map;
-import javax.persistence.Entity;
-import javax.persistence.Table;
+import javax.persistence.*;
 import play.data.validation.Constraints;
+import services.applicant.ApplicantData;
 
 @Entity
-/** The ebeans mapped class that represents an individual applicant */
+/** The ebean mapped class that represents an individual applicant */
 @Table(name = "applicants")
 public class Applicant extends BaseModel {
   private static final long serialVersionUID = 1L;
+  private static final String EMPTY_APPLICANT_DATA_JSON = "{ \"applicant\": {}, \"metadata\": {} }";
+  private ApplicantData applicantData;
 
-  public Map<String, Object> getObject() {
-    return object;
-  }
+  @Constraints.Required @DbJson private String object;
```

Comment:
Suggestion (blocking): please put each annotation [on a line of its own](https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations)

Reply:
Done

Reply:
It looks like our formatter puts these all on the same line. I'll create an issue for updating our formatter if possible:
* annotations on their one lines
* no one-line methods

Reply:
Created issue #93 for the annotations. Turns out we don't actually have one-line methods - that was just a visual collapse IntelliJ was doing.